Swiftfin/Components/ServerTicks.swift

```diff
@@ -0,0 +1,84 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Foundation
+
+struct ServerTicks {
```

Comment:
I have thought previously about making ticks more strongly typed, but as either probably something loose like `typealias Ticks = Int64` or making a stronger type like this conforming as well to `ExpressibleByIntegerLiteral` with necessary functions and values like you've done below.

However, I would want this in the SDK so we can set the type for all `tick` properties during generation.